tsl/test/sql/data_node.sql

```diff
@@ -16,6 +16,16 @@
 \set DN_DBNAME_5 :TEST_DBNAME _5
 \set DN_DBNAME_6 :TEST_DBNAME _6
 
+-- View to see dimension partitions. Note RIGHT JOIN to see that
+-- dimension partitions are cleaned up (deleted) properly.
+CREATE VIEW hypertable_partitions AS
+SELECT table_name, dimension_id, range_start, data_nodes
+FROM _timescaledb_catalog.hypertable h
```

Comment:
Just wondering if it makes sense to have this view available via our catalogs? Might be a good way for users/devs to check the contents.